src/components/Counter/Counter.css

```diff
@@ -0,0 +1,14 @@
+@value breakpoint, fontstack from 'styles/variables.css';
+@value black from theme(colors);
+
+.root {
+  color: black;
+  font-family: fontstack, Arial, sans-serif;
```

Comment:
Even font names with spaces do not need to be quoted. :)
So either you quote them all, or you quote none of them.